pyuvdata/uvbeam/uvbeam.py

```diff
@@ -212,11 +218,13 @@ def __init__(self):
         )
 
         desc = (
-            "Beam basis vector components -- directions for which the "
-            "electric field values are recorded in the pixel coordinate system. "
+            "Beam basis vector components, essentially the mapping between the "
+            "directions that the electrical field values are recorded in to the "
+            "directions aligned with the pixel coordinate system (or azimuth/zenith "
+            "angle for HEALPix beams)."
             'Not required if beam_type is "power". The shape depends on the '
             'pixel_coordinate_system, if it is "healpix", the shape is: '
-            "(Naxes_vec, Ncomponents_vec, Npixels), otherwise it is "
+            "(Naxes_vec, Ncomponents_vec=2, Npixels), otherwise it is "
```

Comment:
Why is Ncomponents_vec=2 fixed to 2 if it is not fixed to two above?

Reply:
I was thinking for that the HEALPix we support Ncomponents_vec will always be 2, but I think that is better shown in the examples you suggested, I'll take it out here.